pkg/frontend/admin_openshiftcluster_kubernetesobjects_test.go

```diff
@@ -225,51 +225,55 @@ func TestValidateAdminKubernetesObjectsNonCustomer(t *testing.T) {
 		{
 			test:      "invalid groupKind",
 			groupKind: "$",
-			namespace: "openshift-ns",
+			namespace: "openshift",
 			name:      "Valid-NAME-01",
 			wantErr:   "400: InvalidParameter: : The provided groupKind '$' is invalid.",
 		},
 		{
 			test:      "forbidden groupKind",
 			groupKind: "Secret",
-			namespace: "openshift-ns",
+			namespace: "openshift",
 			name:      "Valid-NAME-01",
 			wantErr:   "403: Forbidden: : Access to secrets is forbidden.",
 		},
 		{
 			test:      "forbidden groupKind",
 			groupKind: "Anything.oauth.openshift.io",
-			namespace: "openshift-ns",
+			namespace: "openshift",
 			name:      "Valid-NAME-01",
 			wantErr:   "403: Forbidden: : Access to secrets is forbidden.",
 		},
 		{
 			test:      "allowed groupKind on read",
 			groupKind: "ClusterRole.rbac.authorization.k8s.io",
+			namespace: "openshift",
```

Comment:
I think we introduced a bug somewhere and made tests pass by adding a namespace here and in test cases below.

`ClusterRole` is a non-namespaced kind. Geneva action should not require namespace in this case.

Reply:
Since the function we are using `validateAdminKubernetesObjectsNonCustomer()` specified the `namespace` as parameter, so we have to pass a namespace to it (It will still pass " " as the namespace if we don't do so but " " is not included in our hard-coded list and will return errors)

So, we can fix it by adding the " " in our hard-coded list if it's what we want (The previous filtering `IsOpenshift()` did add " ") or update the `validateAdminKubernetesObjectsNonCustomer()` but it should be a separated issue from this PR.